report/sections/Chapter 01 - Introduction/Intro.tex

```diff
@@ -1,22 +1,21 @@
-The Haskell programming language is purely functional\cite{} and builds upon the lambda calculus model of computation \cite{}.
+The Haskell programming language is purely functional\cite{Haskell_landing} and builds upon the lambda calculus model of computation \cite{Haskell_book}.
 Unlike many other programming languages, Haskell can both be interpreted or compiled into an executable binary file.
 It is a statically typed language implementing many mathematical constructs such as currying, set-builder notation, recursive definitions of both types and functions, and more.
 This allows the programmer to create concise solutions that would normally be quite verbose in other programming paradigms.
-Therefore, many companies such as Meta and Galois use Haskell for cryptography, avionics monitoring and spam filtering.
+Therefore, many companies such as Meta and Galois use Haskell for cryptography, avionics monitoring and spam filtering\cite{Haskell_companies}.
 Thus, Haskell and other languages within the functional programming paradigm are useful tools for software engineers.
 
 
 Haskell is taught in the Programming Paradigms course on the 1st semester of the Software Master's Degree at \aau{}.
 Most students attending this course failed in the past.
 This resulted in a survey being sent out, and a student task force being assembled to understand the issues that caused the high failure rate.
-Here, many students mentioned not feeling experienced enough with the programming languages used in the course to teach the functional and logical programming paradigms.
+Here, many students mentioned not feeling experienced enough with the programming languages used in the course to teach the functional and logical programming paradigms. This can be seen from the slide shown in appendix \ref{chap:PP-intro}.
 
 
 Consequently, \aau{} chose to restructure the course entirely.
 Rather than teaching many different programming languages within the functional and logical programming paradigms, the course was restructured to focus only on Haskell.
 Furthermore, the course now features more practical programming exercises with classroom discussion in-between, rather than traditional lectures followed by a separate exercise session.
-Supporting this way of teaching, the course lectures have been moved to a more classical classroom setup.
-By using a shared monitor at each table, multiple students can work together in groups to solve given problem sets.
+Supporting this way of teaching, the course lectures have been moved to a more specialized classroom setup. The classroom has been set up, such that it has tables for all student groups, each having a monitor that can be shared between the students. This allows the student groups to work together on solving problems given in the lecture.
```

Comment:
This is repeated in another PR: https://github.com/chhoumann/p7/pull/89/files#diff-4bd3cb3279b99c83292f57fa731f36763496ed44c97eda5eecbdda11eab5fcbd